test/integration/functions/op.addition.test.ts

```diff
@@ -8,6 +9,12 @@ describe('evaluation of \'+\' like', () => {
     operation: '+',
     aliases: numeric,
     notation: Notation.Infix,
+    config: {
+      type: 'sync',
+      config: {
+        superTypeDiscoverCallback: unknownType => TypeURL.XSD_INTEGER,
```

Comment:
I know I already suggested another name before, but what do you think about this one?
```suggestion
        getSuperType: unknownType => TypeURL.XSD_INTEGER,
```

Reply:
Yes, my bad. Seems like I didn't change this all the way.

Reply:
Do you mean to just edit the variable name or also the type name?

Reply:
Just the key name is fine.

Reply:
Actually, the type name can be simplified to `ISuperTypeCallback`

Reply:
It isn't an interface name though. So I would keep `SuperTypeCallback`. I did change the keyname to `getSuperType`.